spec/ParseUser.spec.js

```diff
@@ -13,6 +13,91 @@ const passwordCrypto = require('../lib/password');
 const Config = require('../lib/Config');
 const cryptoUtils = require('../lib/cryptoUtils');
 
+describe('Parse.User auth after save testing (original object not being retrieved) ', () => {
+  it('check if user is new when using signup or edit with auth login methods', async done => {
+     let counter = 0;
```

Comment:
Please run lint on the test you added to fix the lint issues.

Reply:
is it okay now?